code/modules/clothing/under/jobs/cargo.dm

```diff
@@ -23,7 +23,7 @@
 	icon_state = "cargotech"
 	inhand_icon_state = "lb_suit"
 	body_parts_covered = CHEST|GROIN|ARMS
-	mutantrace_variation = MUTANTRACE_VARIATION
+	supports_variations = DIGITIGRADE_VARIATION_NO_NEW_ICON
```

Comment:
Cargo tech jumpsuits have a digitigrade icon already, wondering why `DIGITIGRADE_VARIATION_NO_NEW_ICON` is used here?

Reply:
They do? Where is it housed so I can implement that properly.